Add documentation on notification presets/format (#16244)
DataDog/documentation: content/en/monitors/notify/_index.md

{{% notifications-integrations %}}

### Toggle additional content

> Providing additional context, monitor notifications can include content such as the monitor's query, the @-mentions used, metric snapshots (for metric monitors), and links back to relevant pages in Datadog. However, you might decide that for some monitors this additional content is not needed. For that, you can decide on each inidividual monitor, which content you would like to strip away from its notifications. The options are:

Reviewer: ```suggestion
Monitor notifications can provide additional context, such as the monitor's query, the @-mentions used, metric snapshots (for metric monitors), and links back to relevant pages in Datadog. You can configure the types of additional context included in each monitor's notifications. The options are:
```

> - **Default**: no content is hidden
> - **Hide Query**: removes the monitor's query from its notification
> - **Hide Handles**: strips away the @-mentions that are used in the notification message
> - **Hide All**: notification does not include query, handles, any snapshot (for metric monitors), and additional links in footers

Reviewer: ```suggestion
- **Default**: No content is hidden.
- **Hide Query**: Remove the monitor's query from the notification message.
- **Hide Handles**: Remove the @-mentions that are used in the notification message.
- **Hide All**: Notification message does not include query, handles, any snapshots (for metric monitors), or additional links in footers.
```

grammar stuff

> Note, some content might not be displayed by defauly depending on the integration.

Reviewer: ```suggestion
**Note**: Depending on the integration, some content may not be displayed by default.
```

{{% notifications-integrations %}}

### Toggle additional content

Monitor notifications include content such as the monitor's query, the @-mentions used, metric snapshots (for metric monitors), and links back to relevant pages in Datadog. You have the option to choose which content you would like to include or exclude from notifications for individual monitors. 

> {{< img src="monitors/notifications/monitor_notification_presets.png" alt="Set a monitor preset" style="width:100%;" >}}

Reviewer 2: ```suggestion
{{< img src="monitors/notifications/monitor_notification_presets.png" alt="Set a monitor preset" style="width:70%;" >}}
```

> {{< img src="monitors/notifications/monitor_notification_presets.png" alt="Set a monitor preset" style="width:70%;" >}}

Reviewer 2: ```suggestion
{{< img src="monitors/notifications/monitor_notification_presets.png" alt="Set a monitor preset" style="width:100%;" >}}
```
